src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md

```diff
@@ -1 +1,5 @@
 # Changelog
+
```

Comment:
Add a header for `Unreleased` to keep track of newer unreleased changes.

Reply:
Fixed with #1197.